crafting-dead-survival/src/main/java/com/craftingdead/survival/CraftingDeadSurvival.java

```diff
@@ -278,14 +282,95 @@ public void handleBiomeLoading(BiomeLoadingEvent event) {
     while (iterator.hasNext()) {
       MobSpawnSettings.SpawnerData spawnEntry = iterator.next();
       if (spawnEntry.type == EntityType.ZOMBIE) {
-        iterator.add(new MobSpawnSettings.SpawnerData(
-            SurvivalEntityTypes.ADVANCED_ZOMBIE.get(), 40, 2, 8));
-        iterator.add(new MobSpawnSettings.SpawnerData(
-            SurvivalEntityTypes.FAST_ZOMBIE.get(), 15, 2, 4));
-        iterator.add(new MobSpawnSettings.SpawnerData(
-            SurvivalEntityTypes.TANK_ZOMBIE.get(), 5, 2, 4));
-        iterator.add(new MobSpawnSettings.SpawnerData(
-            SurvivalEntityTypes.WEAK_ZOMBIE.get(), 30, 2, 12));
+        if (serverConfig.zombiesAdvancedZombie.get()) {
+          iterator.add(new MobSpawnSettings.SpawnerData(
+              SurvivalEntityTypes.ADVANCED_ZOMBIE.get(),
+              serverConfig.zombiesSpawnAdvancedZombieSpawnWeight.get(),
+              serverConfig.zombiesSpawnAdvancedZombieMinSpawn.get(),
+              serverConfig.zombiesSpawnAdvancedZombieMaxSpawn.get()));
+        }
+        if (serverConfig.zombiesFastZombie.get()) {
+          iterator.add(new MobSpawnSettings.SpawnerData(
+              SurvivalEntityTypes.FAST_ZOMBIE.get(),
+              serverConfig.zombiesSpawnFastZombieSpawnWeight.get(),
+              serverConfig.zombiesSpawnFastZombieMinSpawn.get(),
+              serverConfig.zombiesSpawnFastZombieMaxSpawn.get()));
+        }
+        if (serverConfig.zombiesTankZombie.get()) {
+          iterator.add(new MobSpawnSettings.SpawnerData(
+              SurvivalEntityTypes.TANK_ZOMBIE.get(),
+              serverConfig.zombiesSpawnTankZombieSpawnWeight.get(),
+              serverConfig.zombiesSpawnTankZombieMinSpawn.get(),
+              serverConfig.zombiesSpawnTankZombieMaxSpawn.get()));
+        }
+        if (serverConfig.zombiesWeakZombie.get()) {
+          iterator.add(new MobSpawnSettings.SpawnerData(
+              SurvivalEntityTypes.WEAK_ZOMBIE.get(),
+              serverConfig.zombiesSpawnWeakZombieSpawnWeight.get(),
+              serverConfig.zombiesSpawnWeakZombieMinSpawn.get(),
+              serverConfig.zombiesSpawnWeakZombieMaxSpawn.get()));
+        }
+      }
+    }
+  }
+
+  @SubscribeEvent
+  public void handleLivingDrops(LivingDropsEvent event) {
+    if (event.getEntity() instanceof AdvancedZombie &&
+        !CraftingDeadSurvival.serverConfig.zombiesDeathDrops.get()) {
+      event.getDrops().clear();
```

Comment:
Move this into `LivingExtensionImpl#handleDeathLoot`

Reply:
See `com.craftingdead.survival.world.entity.SurvivalPlayerHandler`. You could create a class called `SurvivalLivingHandler` that implements `LivingHandler` which would allow you `handleDeathLoot`.